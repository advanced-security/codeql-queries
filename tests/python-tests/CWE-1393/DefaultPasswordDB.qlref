CWE-1393/DefaultPasswordDB.ql