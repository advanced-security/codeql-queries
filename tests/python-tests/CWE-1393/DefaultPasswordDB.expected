| default_password.py:12:16:12:82 | Attribute() | Default value in security-sensitive database 'db' $@ assigned to variable 'password' | default_password.py:12:16:12:82 | Attribute() | column |
